docs/internals/contributing/committing-code.txt

```diff
@@ -121,9 +121,10 @@ Django's Git repository:
   immediately, so you may have to wait a couple of days before getting a
   response.
 
-* Write detailed commit messages in the past tense, not present tense.
+* Write detailed commit messages in `imperative mood
+  <http://chris.beams.io/posts/git-commit/>`.
 
-  * Good: "Fixed Unicode bug in RSS API."
+  * Good: "Fix Unicode bug in RSS API."
   * Bad: "Fixes Unicode bug in RSS API."
   * Bad: "Fixing Unicode bug in RSS API."
 
```

Comment:
Should `* Bad: "Fixed Unicode bug in RSS API."` be added here as well. It would be drastic change though for the current contributors.